.eslintrc

```diff
@@ -1,42 +1,8 @@
 {
-  "extends": ["airbnb", "prettier"],
```

Comment:
We need prettier config, it works together with the prettier plugin and the plugin in IDEs to match the rules everywhere.

Reply:
I wasn't aware that the IDEs need that. Because the CRA config does not have it I assumed it would not be necessary. I re-added it.